neuralmonkey/decoders/decoder.py

```diff
@@ -43,8 +43,8 @@ def __init__(self,
                      tf.Tensor]]=None,
                  use_attention: bool=False,
                  embeddings_encoder: Optional[Any]=None,
-                 rnn_cell: str='GRU',
                  attention_on_input: bool=True,
+                 rnn_cell: str='GRU',
```

Comment:
vynech to "ááá".. Když jsem viděl, že to má 21 files changed a tohle na mě vyskočilo jako první, tak jsem se zhrozil, že tam bude dvacet souborů podobně upravenejch.. :-)